index.html

```diff
@@ -1,66 +1,88 @@
+    <title>Sprint Challenge - Home</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
     <div class="container">
-   
+
+        <nav class="menu">
+            <img src="img/lambda-black.png" alt="Lambda">
+            <a href="services.html">Services</a>
+            <a href="">Product</a>
+            <a href="">Vision</a>
+            <a href="">Features</a>
+            <a href="about.html">About</a>
+            <a href="">Contact</a>
+        </nav>
+
+        <div>
+            <img class="top-image" src="img/jumbo.jpg" alt="Jumbo lights">
+        </div>
+
         <section class="top-content">
             <div class="text-container">
-                <h2>The Future</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <h2>The Future</h2><br>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                    efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                    tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
             <div class="text-container">
                 <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
+                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt
+                    efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas
+                    tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box1">Box 1</div>
```

Comment:
This is the most ideal way to handle the various boxes. Great job avoiding the use of an 'id'.